Cmake 3.28 issue

- axmol version: 
- devices test on:
- developing environments
   - NDK version: r23c
   - Xcode version: 14.2
   - Visual Studio: 
      - VS version: 2019 (16.11), 2022 (17.6+)
      - MSVC version: 1929, 1934, 19.35, 19.36, 19.37
      - Windows SDK version: 10.0.22621.0
   - cmake version: 
Steps to Reproduce:

Using `cmake -S . -B build_x64 -G "Visual Studio 17 2022" -A x64` results in this error:

```
CMake axmol target_precompile_headers
CMake Error at G:/dev/axmol/core/CMakeLists.txt:355 (exec_program):
  The exec_program command should not be called; see CMP0153.  Use
  execute_process() instead.
```

[CMP0153](https://cmake.org/cmake/help/latest/policy/CMP0153.html)

This only occurs if the `CMakeLists.txt` has the minimum cmake required set to 3.28 via `cmake_minimum_required(VERSION 3.28)`

--- a/core/CMakeLists.txt
+++ b/core/CMakeLists.txt
@@ -342,18 +342,18 @@ if(WINDOWS)
     if(WINDOWS_STORE)
         file(TO_NATIVE_PATH ${CMAKE_BINARY_DIR} AX_TOP_SOLUTION_DIR)
         configure_file(${_AX_ROOT}/cmake/CppWinRT.props.in ${CMAKE_BINARY_DIR}/CppWinRT.props @ONLY)
-       
+
         # intsall cppwinrt package for xaml apps
-        exec_program(${NUGET_EXE}
-                ARGS install "Microsoft.Windows.CppWinRT" -Version ${AX_CPPWINRT_VERISON} -ExcludeVersion -OutputDirectory "\"${CMAKE_BINARY_DIR}/packages\"")
+        execute_process(COMMAND "${NUGET_EXE}"
+                install "Microsoft.Windows.CppWinRT" -Version ${AX_CPPWINRT_VERISON} -ExcludeVersion -OutputDirectory "${CMAKE_BINARY_DIR}/packages")
         set_target_properties(${_AX_CORE_LIB} PROPERTIES
             VS_PROJECT_IMPORT ${CMAKE_BINARY_DIR}/packages/Microsoft.Windows.CppWinRT/build/native/Microsoft.Windows.CppWinRT.props
         )
         target_link_libraries(${_AX_CORE_LIB} ${CMAKE_BINARY_DIR}/packages/Microsoft.Windows.CppWinRT/build/native/Microsoft.Windows.CppWinRT.targets)
     endif()
     if(AX_ENABLE_MSEDGE_WEBVIEW2)
-        exec_program(${NUGET_EXE}
-            ARGS install "Microsoft.Web.WebView2" -Version 1.0.992.28 -ExcludeVersion -OutputDirectory "\"${CMAKE_BINARY_DIR}/packages\"")
+        execute_process(COMMAND "${NUGET_EXE}"
+            install "Microsoft.Web.WebView2" -Version 1.0.992.28 -ExcludeVersion -OutputDirectory "${CMAKE_BINARY_DIR}/packages")
 
         if(CMAKE_GENERATOR MATCHES "Ninja")
             target_link_libraries(${_AX_CORE_LIB} ${CMAKE_BINARY_DIR}/packages/Microsoft.Web.WebView2/build/native/${ARCH_ALIAS}/WebView2Loader.dll.lib)

